### PR TITLE
Update unit choices

### PR DESCRIPTION
**Pull request by kevincarrogan**
Update the available units for products

### CODE REVIEW

File: api/applications/migrations/0059_alter_goodonapplication_unit.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    ("TON", "Tonnes"),
+                    ("KGM", "Kilograms"),
+                    ("GRM", "Grams"),
+                    ("MIM", "Milligrams"),
+                    ("MCM", "Micrograms"),
+                    ("MTR", "Metres"),
+                    ("MTK", "Square metres"),
+                    ("MTQ", "Cubic metres"),
+                    ("LTR", "Litres"),
+                    ("MIR", "Millilitres"),
```
Comment (currycoder) [inline api/applications/migrations/0059_alter_goodonapplication_unit.py:29]: Where did these short names come from out of interest?

File: api/staticdata/units/enums.py [hunk 2]
```diff
@@ -1,36 +1,28 @@
 class Units:
-    """
-    LITE Frontend can dynamically pluralise these
-    units, to enable this include an (s) at the
-    end of the unit.
```
Comment (currycoder) [inline api/staticdata/units/enums.py:5]: This intrigues me

File: api/staticdata/units/enums.py [hunk 3]
```diff
@@ -1,36 +1,28 @@
...
-    """
-
     GRM = "GRM"
     KGM = "KGM"
     NAR = "NAR"
     MTK = "MTK"
     MTR = "MTR"
     LTR = "LTR"
     MTQ = "MTQ"
-    ITG = "ITG"
```
Comment (currycoder) [inline api/staticdata/units/enums.py:15]: Do we have any existing ITG goods?

File: api/staticdata/units/enums.py (hunk 2 above)
Comment (kevincarrogan) [reply api/staticdata/units/enums.py:5]: It doesn't work, hence me removing this comment.

File: api/applications/migrations/0059_alter_goodonapplication_unit.py (hunk 1 above)
Comment (kevincarrogan) [reply api/applications/migrations/0059_alter_goodonapplication_unit.py:29]: That's what's always been there. I was reluctant to change it.

File: api/staticdata/units/enums.py (hunk 3 above)
Comment (kevincarrogan) [reply api/staticdata/units/enums.py:15]: Nope.

**Review by currycoder — APPROVED** [review]

**Pull request merged**